src/pages/Home/styles.ts

```diff
@@ -38,6 +38,7 @@ export const Products = styled.div`
 `;
 
 export const Product = styled.div`
+  border: 2px solid var(--primary);
```

Comment:
Não curti muito essa borda, como tinha falado no grupo. Estuda ai colocar ela apenas no hover.

Reply:
Isso vai ser mudado ainda. To vendo um Layout melhor